patches/minecraft/net/minecraft/block/BlockRedstoneComparator.java.patch

```diff
@@ -7,7 +7,7 @@
 +    @Override
 +    public void onNeighborChange(IBlockAccess world, BlockPos pos, BlockPos neighbor)
 +    {
-+        if (pos.func_177956_o() == neighbor.func_177956_o() && world instanceof World)
++        if (pos.func_177956_o() == neighbor.func_177956_o() && world instanceof net.minecraft.world.WorldServer)
```

Comment:
Actually thats not the case, there are some WorldServers that are remote due to mods using them to simulate things.